lib/Page.js

```diff
@@ -26,7 +26,7 @@ const {Keyboard, Mouse, Touchscreen} = require('./Input');
 const Tracing = require('./Tracing');
 const {helper, debugError, assert} = require('./helper');
 const {Coverage} = require('./Coverage');
-const {Worker} = require('./Worker');
+const {Worker: PuppeteerWorker} = require('./Worker');
```

Comment:
@mathiasbynens Can the export of `Worker` be renamed to `PuppeteerWorker` in `Worker.js` or does that constitute a breaking change?

Reply:
I think that would be a breaking change if we support people who have done:

```js
const { Worker } = require('puppeteer/lib/Worker')
```

I'm not sure how common that is - @mathiasbynens WDYT?

Reply:
It's a potentially breaking change that we can avoid in this case, so let's just stick to the more conservative change for now.